### PR TITLE
Shrink the size of the login/logout area of the main activity

### PR DESCRIPTION
**Pull request by lack**
This seemed overly large to me - This new layout still has the login area
obvious, but not overpowering.

Signed-off-by: Jim Ramsay <i.am@jimramsay.com>

### CODE REVIEW

**Comment by oryanm** [comment]
The login area is intentionally set to be in a fixed 1/1.6 ratio with the settings area.
I don't really see a reason to change that.

**Comment by lack** [comment]
I don't mind the large size when the user isn't logged in yet.  But when the login is successful, it seems a waste of screen real estate, when all that I care about after I log in is that (1) I'm still logged in and (2) access to the settings below.

**Comment by lack** [comment]
Here's an example of what I see on my widget settings screen:

image

**Comment by lack** [comment]
Here's what it looks like after my change:

image

**Comment by lack** [comment]
I've reworked this a bit again.  It leaves the "Login" area about the same size, but shrinks down that area after you've logged in.

Here's before you login, with this change:

image

Here's after you login, with this change:

image

Compare this to the way it used to be before this change - When you're logged in there's a lot of screen real estate dedicated to empty space where we could be showing more preferences:

image

File: app/src/main/res/layout/fragment_logged_in.xml [hunk 1]
```diff
@@ -11,27 +11,28 @@
...
-            android:textAppearance="?android:attr/textAppearanceLarge" />
+            android:textAppearance="?android:attr/textAppearanceLarge"
+            android:layout_weight="1" />
 
         <Button
             android:id="@+id/logout_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_gravity="end|bottom"
+            android:layout_gravity="bottom|end"
```
Comment (oryanm) [inline app/src/main/res/layout/fragment_logged_in.xml]: Why was this changed?

File: app/src/main/res/layout/fragment_logged_in.xml [hunk 2]
```diff
@@ -11,27 +11,28 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center" />
 
-    <FrameLayout
-        android:id="@+id/signed_panel"
+    <LinearLayout
+        android:orientation="horizontal"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:visibility="gone">
```
Comment (oryanm) [inline app/src/main/res/layout/fragment_logged_in.xml:18]: Why remove the visibility attribute?

File: app/src/main/res/layout/fragment_logged_in.xml (hunk 1 above)
Comment (lack) [reply app/src/main/res/layout/fragment_logged_in.xml]: It was reordered by the IDE.  No idea why :)

I guess it likes them in alphabetical order?

File: app/src/main/res/layout/fragment_logged_in.xml (hunk 2 above)
Comment (lack) [reply app/src/main/res/layout/fragment_logged_in.xml:18]: Good catch.  This should not be in here.

Comment (lack) [reply app/src/main/res/layout/fragment_logged_in.xml:18]: Fixed.

File: app/src/main/res/layout/fragment_logged_in.xml (hunk 1 above)
Comment (oryanm) [reply app/src/main/res/layout/fragment_logged_in.xml]: Interesting. Mine doesn't do that.

**Review by oryanm — COMMENTED** [review]
OK I've pushed this change manually with slight changes. So we can close this PR.
